packages/riipen-ui/src/components/Accordion.jsx

```diff
@@ -0,0 +1,120 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ThemeContext from "../styles/ThemeContext";
+
+import AccordionContext from "./AccordionContext";
+import Collapse from "./Collapse";
+
+const Accordion = props => {
+  const {
+    children: childrenProp,
+    classes,
+    defaultExpanded = false,
+    disabled = false,
+    ...other
+  } = props;
+
+  const theme = React.useContext(ThemeContext);
+
+  const [expanded, setExpanded] = React.useState(defaultExpanded);
+
+  const handleChange = React.useCallback(() => {
+    setExpanded(!expanded);
+  }, [expanded, setExpanded]);
+
+  const [summary, ...children] = React.Children.toArray(childrenProp);
+
+  const contextValue = React.useMemo(
+    () => ({ disabled, expanded, toggle: handleChange }),
+    [expanded, disabled, handleChange]
+  );
```

Comment:
fancy

Reply:
I have very little understanding of this. #copypasta